web/src/components/core/IssuesDialog.jsx

```diff
@@ -162,23 +161,29 @@ export default function IssuesPage() {
   }, [client, cancellablePromise, setIsLoading]);
 
   const update = useCallback((issues) => {
-    setIssues(current => ({ ...current, ...issues }));
-    if (notification.issues) updateNotification({ issues: false });
-  }, [notification, setIssues, updateNotification]);
+    setIssues(current => ([...current, ...(issues[sectionId] || [])]));
+  }, [setIssues, sectionId]);
 
   useEffect(() => {
     load().then(update);
     return client.onIssuesChange(update);
   }, [client, load, update]);
 
   return (
-    // TRANSLATORS: page title
-    <Page icon="problem" title="Issues">
+    <Popup
+      isOpen={isOpen}
+      title={titles[sectionId].text}
+      titleIconVariant={titles[sectionId].icon}
+      data-content="issues-summary"
+    >
       <If
         condition={isLoading}
-        then={<SectionSkeleton numRows={4} />}
+        then={<Icon name="loading" className="icon-big" />}
```

Comment:
As said, just render the `<Section>` here. `IssuesContent` internal component it's no longer needed. And maybe place the loading inside the section, 

```jsx

<Section aria-label="...">
  <If condition={isLoading} then={...} />
  <IssueItems issues={issues} />
</Section>
```

Or a better approach if you found it while coding :)